lib/TextOperations.cpp

```diff
@@ -210,4 +248,28 @@ std::string TextOperations::escapeString(std::string input)
 	return input;
 }
 
+int TextOperations::extractNumberAfterHash(const std::string & input)
+{
+    size_t hashPos = input.find('#');
+	if(hashPos != std::string::npos)
+		return -1;

```

Comment:
So function extracts only non-negative number, not a number

Reply:
/// extracts the number after a # character (useful in autosave filenames)

I think should be obvious without renaming the function. If any suggestion I can rename.